Resources/Prototypes/Catalog/Fills/Crates/security.yml

```
@@ -114,24 +114,24 @@
-    - id: ClothingOuterHardsuitSyndieReverseEngineered
-
-- type: entity
-  id: CrateSecurityGuanYuTacsuit
-  parent: CrateSecgear
-  name: shanlin tacsuit crate
-  description: Contains a single CSA-80UA Guan-Yu tacsuit. Requires Security access to open.
-  components:
-  - type: StorageFill
-    contents:
-    - id: ClothingOuterHardsuitJuggernautReverseEngineered
+#- type: entity
```

yamllint

[warning] missing starting space in comment

(comments)